samples/SynchronizeSamples/info.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<info:toolkitInfoModel xmlns:common="http://www.ibm.com/xmlns/prod/streams/spl/common" xmlns:info="http://www.ibm.com/xmlns/prod/streams/spl/toolkitInfo">
+  <info:identity>
+    <info:name>SynchronizeSamples</info:name>
```

Comment:
Can we name this samples toolkit `synchronize` to be consistent with `redundant`, `shedding` and `switches`?